README.md

```diff
@@ -1,14 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# My Portfolio
+# Portfolio.
 
-> My Microverse portfolio Desktop Version Final
+> In this branch (animation_transition) I used the Animation and transition to build attractive web design I did it with my collaborators and you can check the commits of my collaborators at this branch.
 
 ## Built With
 
-- HTML/CSS
+- HTML
+- CSS
+- Flexbox
 - Figma
--
+- Formspree service
+- (RWD) responsive web design using media query
+- Animation and Transition
+
+
+
+## Live Demo 
+
+- Gain access to this awesom project by :
+- heading over to "https://github.com/mrkamin/mrkamin.github.io" to view the Github repo.
+- Forking it to your Github account (You can as well "clone" it to your local machine )
+And there you have it all running in your local machine 😎
+
+### Links
+
+- Solution URL: [https://github.com/mrkamin/mrkamin.github.io]
+- Live Site URL: [https://mrkamin.github.io/]
```

Comment:
Great job @mrkamin 🤝

- [ ] [Optional] Kindly use [markdown syntax](https://www.markdownguide.org/basic-syntax/) to organize the content of the readme file.